src/drivers/intel/cavs/hda-dma.c

```diff
@@ -161,7 +161,7 @@ struct hda_chan_data {
 
 	uint32_t period_bytes;
 	uint32_t buffer_bytes;
-	struct work dma_ch_work;
+	struct ll_task dma_ch_work;
```

Comment:
This should be "struct task" (and be used by all client scheduler API calls) regardless of scheduling type.

Reply:
On PR's HEAD it is done